src/lib/select/select.ts

```diff
@@ -349,6 +352,8 @@ export class MdSelect implements AfterContentInit, ControlValueAccessor, OnDestr
     } else {
       this.onClose.emit();
     }
+
+    this._panelDoneAnimating = true;
```

Comment:
It seems like this will only work the first time you open the select.  Don't you need to reset this to false when the panel closes?